composer/spec/dependabot/composer/update_checker/latest_version_finder_spec.rb

```diff
@@ -165,7 +165,7 @@
             package_manager: "composer"
           )
         end
-        it { is_expected.to eq(Gem::Version.new("2.3.x.pre.dev")) }
+        it { is_expected.to eq(Gem::Version.new("2.3.0-RC4")) }
```

Comment:
This is the impact of dropping support for dev releases, but as I mentioned in the PR/commit description:
> Because pre-release version suffixes such as `BETA1` or
   `-RC1` are still returned on the main channel, I don't think we
   should invest any effort into supporting dev releases. They're not
   used much, they'd be another thing to support, and the pre-release
   suffixes provide a good-enough replacement.